src/pcapi/core/users/models.py

```diff
@@ -181,22 +180,17 @@ def checkPassword(self, passwordToCheck):
     def get_id(self):
         return str(self.id)
 
-    def hasRights(self, rights, offerer_id):
+    def hasRights(self, offerer_id):
```

Comment:
Est-ce qu'on ne supprimerait pas complètement cette notion de droits ? :)
```suggestion
    def has_access(self, offerer_id):
```